test/dynamo/test_misc.py

```diff
@@ -79,6 +79,7 @@
 from torch.testing._internal.common_utils import (
     freeze_rng_state,
     IS_FBCODE,
+    scoped_load_inline,
```

Comment:
@xmfan can you give more context about what this PR is (and put it into the PR body?)

Reply:
It sounds like this PR is supposed to fix flakiness, do you know what caused the flakyness?